Zend/zend_vm_execute.h

```diff
@@ -4143,7 +4112,7 @@ static ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL ZEND_FE_RESET_RW_SPEC_CONST_HANDLER
 
 static ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL ZEND_EXIT_SPEC_CONST_HANDLER(ZEND_OPCODE_HANDLER_ARGS)
 {
-#if 0 || (IS_CONST != IS_UNUSED)
```

Comment:
This is generated code, it should not be changed (or only via zend_vm_def.h). Otherwise these `0 ||` will just reappear the next time someone regenerates the VM.